dg645Sup/dg645.db

```diff
@@ -0,0 +1,308 @@
+record(bo, "$(P)$(R)SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM=0)")
+}
+
+record(bo, "$(P)$(R)DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE=0)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+## Trigger settings and readback + suplementary records for reading
+## T0 and T1 that are not implemented by Delaygen
+
+# Trigger set
+
+record(mbbo,"$(P)$(R)TriggerSource:SP")

```

Comment:
Our standards specify that we should be using uppercase PV names throughout. We can't help the delaygen module, but our own DBs we should enforce those standards

Reply:
All of the PV names were converted to follow our standards, with exception of 2 records that serve as extension of delaygen module so they can work well with delaygen.